client/src/js/modules/dc/views/gridplot.js

```diff
@@ -88,6 +88,7 @@ define(['jquery', 'marionette',
             this.attachments = new Attachments()
             this.attachments.queryParams.id = this.getOption('ID')
             this.attachments.queryParams.filetype = 'pia'
+            this.attachments.setSorting('DATACOLLECTIONFILEATTACHMENTID', 1)
```

Comment:
Does this work because pia_estimated_d_min is always created last is there a chance that it is not, in that case is it important?